group.py

```diff
@@ -2,4 +2,49 @@
 
 # Your code to go here...
 
-my_group =
+class person:
+
+    def __init__(self,name,age,job=''):
+        self.name = name
+        self.age = age
+        self.job = job
+        self.relations = {}
+
+    def relationship(self, person2, relationship):
+        self.relations[person2] = relationship
+
+def max_age(group):
+    ### Takes an input of a list of names in the class "person" and outputs the maximum age in the group ###
+    return max([name.age for name in group])
+
+def mean_age(group):
+    ### Takes an input of a list of names in the class "person" and outputs the mean age in the group ###
+    return sum([name.age for name in group])/len(group)
+
+def max_age_1_relation(group):
+    ### Takes an input of a list of names in the class "person" and outputs the max age in the group if they have at least one relation ###
+    return max([name.age for name in group if len(name.relations) > 0])
```

Comment:
Nice, could also do this to use the built in falsy-ness of empty dictionaries
```suggestion
    return max([name.age for name in group if name.relations])
```